specification/sql/resource-manager/Microsoft.Sql/preview/2024-05-01-preview/InstancePoolOperations.json

```diff
@@ -0,0 +1,268 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/instancePools/{instancePoolName}/operations": {
+      "get": {
+        "tags": [
+          "InstancePoolOperations"
+        ],
+        "description": "Gets a list of operations performed on the instance pool.",
+        "operationId": "InstancePoolOperations_ListByInstancePool",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
```

Comment:
>ResourceGroupParameter [](http://example.com/codeflow?start=63&length=22)

Why are you not using the parameters defined in common-types? [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json)

Reply:
I just found an older post where SQL API team said we are not onboarded to v5/types. Please ping SQL API ARM team if onboarding to v5/types is needed.